wazo_confgend/generators/tests/test_extensionsconf.py

```diff
@@ -23,9 +23,11 @@ def setUp(self):
         self.hint_generator = Mock(HintGenerator)
         self.tpl_mapping = {}
         self.tpl_helper = TemplateHelper(DictLoader(self.tpl_mapping))
-        self.extensionsconf = ExtensionsConf('etc/wazo-confgend/templates/contexts.conf',
-                                             self.hint_generator, self.tpl_helper)
+        self.extensionsconf = ExtensionsConf(
+            'etc/wazo-confgend/templates/contexts.conf', self.hint_generator, self.tpl_helper
+        )
         self.output = StringIO()
+        self.maxDiff = 10000
```

Comment:
Unused variable
```suggestion
```

Reply:
It's not unused, it's a [setting for unittest](https://docs.python.org/3/library/unittest.html#unittest.TestCase.maxDiff). 
I can remove it, but I increased the limit because otherwise it won't show the diff on configs in tests because they're too long. Do you still want me to remove it?

Reply:
Oh I did not know this option. Thank you
Can we set this param on the class attribute like:
```python
class TestExtensionsConf(unittest.TestCase):
    maxDiff = 10000

    def setUp(self):
```
It will make a little bit clearer that is a setting for the whole class

Reply:
Oh yeah, great idea. I'll do that.